libs/langchain/langchain/callbacks/openai_info.py

```diff
@@ -82,11 +91,14 @@ def standardize_model_name(
     """
     model_name = model_name.lower()
     if "ft-" in model_name:
-        return model_name.split(":")[0] + "-finetuned"
+        return model_name.split(":")[0] + "-finetuned-legacy"
+    if "ft:" in model_name:
+        return model_name.split(":")[1] + "-finetuned"
```

Comment:
Does this work on Azure OpenAI? It looks like Azure OpenAI fine-tuned models have the following structure
`<model_name>.ft-<job_id>` ([reference](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/fine-tuning?tabs=turbo&pivots=programming-language-studio)), which would go into the previous `if` case but without doing the right thing I fear :/

Reply:
youre right, Ill open a new PR about the azure fix, Ill tag you 💪